aspnetcore/getting-started/index.md

```diff
@@ -56,99 +75,36 @@ This document provides steps for creating and running an ASP.NET Core app.
    
 ---
 
-4. Run the app:
-
-   ```console
-   cd aspnetcoreapp
-   dotnet run
-   ```
-
-5. Browse to [http://localhost:5001](http://localhost:5001).  Click **Accept** to accept the privacy and cookie policy. This app doesn't keep personal information.
-
-6. Open *Pages/About.cshtml* and modify the page with the following highlighted markup:
-
-   [!code-cshtml[](sample/getting-started/about.cshtml?highlight=9)]
-
-7. Browse to [http://localhost:5001/About](http://localhost:5001/About) and verify the changes are displayed.
-
-[!INCLUDE [next steps](~/includes/getting-started/next-steps.md)]
-
-::: moniker-end
-
-::: moniker range="= aspnetcore-2.0"
-
-1. Install the [!INCLUDE [](~/includes/net-core-sdk-download-link.md)].
-
-2. Create a new ASP.NET Core project.
-
-   Open a command shell. Enter the following command:
-
-   ```console
-   dotnet new razor -o aspnetcoreapp
-   ```
+## Run the app
 
-3. Run the app with the following commands:
+* Run the following commands:
 
    ```console
    cd aspnetcoreapp
    dotnet run
    ```
 
-4. Browse to [http://localhost:5000](http://localhost:5000).
-
-5. Open *Pages/About.cshtml* and modify the page to display the message "Hello, world! The time on the server is @DateTime.Now":
-
-   [!code-cshtml[](sample/getting-started/about.cshtml?highlight=9&range=1-9)]
-
-6. Browse to [http://localhost:5000/About](http://localhost:5000/About) and verify the changes.
-
-[!INCLUDE [next steps](~/includes/getting-started/next-steps.md)]
-
-::: moniker-end
-
-::: moniker range="<= aspnetcore-1.1"
-
-1. Install the .NET Core **SDK Installer** for SDK 1.0.4 from the [.NET Core All Downloads page](https://www.microsoft.com/net/download/all).
-
-2. Create a folder for a new ASP.NET Core project.
-
-   Open a command shell. Enter the following commands:
-
-   ```console
-   mkdir aspnetcoreapp
-   cd aspnetcoreapp
-   ```
-
-3. If you have installed a later SDK version on your machine, create a *global.json* file to select the 1.0.4 SDK.
-
-   ```json
-   {
-     "sdk": { "version": "1.0.4" }
-   }
-   ```
+* Browse to [http://localhost:5001](http://localhost:5001).  Click **Accept** to accept the privacy and cookie policy. This app doesn't keep personal information.
```

Comment:
- There are 2 spaces after the 1st sentence.
- http --> https (this way, the user understands why they trusted the dev cert)